app/controllers/concerns/application_shared.rb

```diff
@@ -15,4 +15,12 @@ def set_timezone
     # Reset timezone for the next thread
     Time.zone = default_timezone
   end
-end
\ No newline at end of file
+
+  private
+
```

Comment:
I can't dynamically create a method `#{controller_name}_params` in a concern, as the concern will try to define the method name before being called from a subclass context (confusing I know).

Basically this will not work:
```ruby
define_method "#{controller_name}_params" do
end
```

I could possibly get a list of `strong_params` compatible controllers and then call `define_method` on all of them, but it feels like more work to maintain.